docs/core/deploying/index.md

```diff
@@ -1,95 +1,143 @@
+title: Application publishing
+description: Learn about the ways to publish a .NET Core application. .NET Core can publish platform-specific or cross-platform apps. You can publish an app as self-contained or as runtime-dependent. Each mode affects how a user runs your app.
+ms.date: 01/31/2020
 ---
-# .NET Core application deployment
+# .NET Core application publishing overview
 
-You can create three types of deployments for .NET Core applications:
+Applications you create with .NET Core can be published in two different modes, and depending on the mode, affects how a user runs your app. Publishing your app self-contained produces an application that contains the .NET Core runtime and libraries, and your application and its dependencies. Publishing your app runtime-dependent produces an application that contains only your application and its dependencies, and users of your application will need to separately install the .NET Core runtime. Both types of applications include a platform-specific executable by default. Runtime-dependent applications without an executable can be created and these applications are cross-platform. 
 
-- Framework-dependent deployment. As the name implies, framework-dependent deployment (FDD) relies on the presence of a shared system-wide version of .NET Core on the target system. Because .NET Core is already present, your app is also portable between installations of .NET Core. Your app contains only its own code and any third-party dependencies that are outside of the .NET Core libraries. FDDs contain *.dll* files that can be launched by using the [dotnet utility](../tools/dotnet.md) from the command line. For example, `dotnet app.dll` runs an application named `app`.
+The following table outlines the basic commands used to publish an app as runtime-dependent or self-contained, per SDK:
 
-- Self-contained deployment. Unlike FDD, a self-contained deployment (SCD) doesn't rely on the presence of shared components on the target system. All components, including both the .NET Core libraries and the .NET Core runtime, are included with the application and are isolated from other .NET Core applications. SCDs include an executable (such as *app.exe* on Windows platforms for an application named `app`), which is a renamed version of the platform-specific .NET Core host, and a *.dll* file (such as *app.dll*), which is the actual application.
+| Type                                                                  | SDK 3.1 | SDK 3.0 | SDK 2.1 | Command | 
+| --------------------------------------------------------------------- | --------| ------- | ------- | ------- |
+| [runtime-dependent executable](#publish-runtime-dependent)            | ✔️       | ✔️       |         | [`dotnet publish`](../tools/dotnet-publish.md) |
+| [runtime-dependent executable](#publish-runtime-dependent)            | ✔️       | ✔️       |         | [`dotnet publish -r <RID> --self-contained false`](../tools/dotnet-publish.md) |
```

Comment:
There are two rows with the same text in the type column.  Can we differentiate them, as in 
* ... for the current runtime
* ... for a specified runtime